cheatsheets_draft/Multifactor_Authentication_Cheat_Sheet.md

```diff
@@ -0,0 +1,327 @@
+
+Some suggestions of possible methods include:
+
+- Providing the user with a number of single-use recovery codes when they first setup MFA.
+- Requiring the user to setup multiple types of MFA (such as a digital certificate, OTP core and phone number for SMS), so that they are unlikely to lose access to all of them at once.
+- Posting a one-use recovery code (or new hardware token) to the user.
+- Requiring the user contact the support team and having a rigorous process in place to verify their identity.
+- Requiring another trusted user to vouch for them.
+
+# Something You Know
+
+The most common type of authentication is based on something the users knows - typically a password. The biggest advantage of this factor is that it has very low requirements for both the developers and the end user, as it does not require any special hardware, or integration with other services.
+
+## Passwords and PINs
+
+Passwords and PINs are the most common form of authentication due to the simplicity of implementing them. The [Authentication Cheat Sheet](Authentication_Cheat_Sheet.md#implement-proper-password-strength-controls) has guidance on how to implement a strong password policy, and the [Password Storage Cheat Sheet](Password_Storage_Cheat_Sheet.md) has guidance on how to securely store passwords.
+
+Most multifactor authentication systems make use of a password, as well as at least one other factor.
+
+It should be noted that PINs, "secret words" and other similar type of information are all effectively the same as passwords. Using two different types of passwords **does not constitute MFA**.
+
+### Pros
+
+- Simple and well understood.
+- Native support in every authentication framework.
+- Easy to implement.
+
+### Cons
+
+- Users are prone to choosing weak passwords.
+- Passwords are commonly re-used between systems.
+
+## Security Questions
+
+Security questions require the user to choose (or create) a number of questions that only they will know the answer to. These are effectively the same as passwords, although they are generally considered weaker. The [Choosing and Using Security Questions Cheat Sheet](Choosing_and_Using_Security_Questions_Cheat_Sheet.md) contains further guidance on how to implement these securely.
+
+### Pros
+
+- Simple and well understood.
+
+### Cons
+
+- Questions often have easily guessable answers.
+- Answers to questions can often be obtained from social media or other sources.
+- Questions must be carefully chosen so that users will remember answers years later.
+
+# Something You Have
+
+The second factor is something that the users possesses. This could be a physical item (such as a hardware token), a digital item (such as a certificate or private key), or based on the ownership of a mobile phone (such as SMS or a software token installed on the phone).
```

Comment:
Some websites use the email to login users. So no passwords, just emails sent to authenticate a user. I think this can fit in here not only as a 2FA, but as a password replacement.

Reply:
I've added a note about this in bf6a9e0422abfc818fcf91f0ed7b13d38cefdecd - is that the sort of thing you were thinking?

Reply:
@ThunderSon are you happy with this change?